### PR TITLE
Some functions required for WebGL

### PR DESCRIPTION
**Pull request by evilpie**
This is probably not perfect we just started learning Reason/Ocaml, but those functions are useful for setting up a minimal WebGL example. We would appreciate your feedback.

### CODE REVIEW

**Comment by bsansouci** [comment]
@evilpie Hey! Nice PR. I don't know what @chenglou thinks about it, but in the meantime you can checkout [regl](https://github.com/bsansouci/regl) which wraps all of those for both OpenGL and WebGL :D

File: src/reasonJs.re [hunk 1]
```diff
@@ -4,6 +4,7 @@ module Window = {
   external innerWidth : t => int = "innerWidth" [@@bs.get];
   external innerHeight : t => int = "innerHeight" [@@bs.get];
   external addEventListener : t => string => (unit => unit) => unit = "addEventListener" [@@bs.send];
+  external requestAnimationFrame : t => (unit => unit) => unit = "requestAnimationFrame" [@@bs.send];
```
Comment (chenglou) [inline src/reasonJs.re]: Let's put this outside, standalone like setTimeout. This is available on server-side without window.

File: src/reasonJs.re [hunk 2]
```diff
@@ -22,12 +23,145 @@ module Math = {
...
 module Document = {
   type element;
   external getElementById : string => element = "document.getElementById" [@@bs.val];
   external appendChild : element => 'element => unit = "appendChild" [@@bs.send];
+
+  /* Should be on CanvasElement */
+  external getContext : element => string => GL.glT = "getContext" [@@bs.send];
 };
 
+external log: string => unit = "console.log" [@@bs.val];
```
Comment (chenglou) [inline src/reasonJs.re]: No need for this! `Js.log` and `print_endline` work =)

**Comment by chenglou** [comment]
Since there aren't big changes needed, I'll merge this and make the two minor corrections. Thanks!

**Comment by chenglou** [comment]
Btw @evilpie: just ran this and BS caught an `external` bug: `vertexAttribPointer` should be passed a `Js.boolean`, not a bool. I'm fixing this.

**Comment by evilpie** [comment]
@chenglou Thanks for merging, I almost forgot about this. I think in the future it would be wise to generate the bindings directly from the WebIDL files.

**Pull request merged**